docs/version0.27.md

```diff
@@ -55,4 +56,8 @@ For more information about this type of operation, see [GC copy forward operatio
 
 You can revert to the behavior in earlier releases by setting [`-Xgc:breadthFirstScanOrdering`](xgc.md#breadthfirstscanordering) when you start your application.
 
+### Stop parsing the JAVA_OPTIONS environment variable
+
+The 0.24 release starting parsing the JAVA_OPTIONS environment variable. This was added in error and has been removed.
```

Comment:
Is it worth mentioning `_JAVA_OPTIONS` here?

Reply:
I guess we should. I wanted to keep it low key since Hotspot doesn't document the option.